### DISCUSSION TITLE
Add a Calendar View

### DISCUSSION THREAD

**Discussion by vHanda** [GitJournal/GitJournal#829] [Feature Requests]
While browsing notes, especially journal entries, having a calendar visible where you can see what dates have entires and jump to them is very useful.

For now, maybe this can just be another view like the Journal / Standard View.

Note to self: Use - https://pub.dev/packages/table_calendar

---
> Please vote on this issue with a 👍reaction. It's one of the primary mechanisms used to prioritize issues.

**Comment by alensiljak** [comment]
You added so many useful features. I can't help but notice that many of these come from Org Mode. When (or rather IF) you have some time, try out Orgzly or https://organice.200ok.ch, or any other Org Mode implementation. There are some fantastic features in there but the implementations are usually lacking, unfortunately. 
Calendar, agenda, todo, schedules, deadlines, events and reminders, progress tracking [1/3] and [60%] formats, collapsing, narrowing, and plenty more.

**Comment by tionis** [comment]
In the same manner, some shortcuts to jump to the note of today or a widget to show it in the home screen would be awesome.
